docs/sdk/typescript.mdx

```diff
@@ -45,18 +42,25 @@ const keypair = Keypair.fromMnemonicSet(mnemonic)[0]
 ## Use the SDK with the Keypair
 
 ### Get the balance
+Use a publicKey to check your Kin balance.
 
 ```ts
 const balance = await sdk.balance(keypair.publicKey)
 ```
 
 ### Request an Airdrop
+Send Kin to a publicKey on the Solana devnet.
+
+Airdrops take some time to complete as we make sure the transaction has been confirmed on the blockchain before returning a response. 
 
 ```ts
 const signature = await sdk.requestAirdrop(keypair.publicKey, "42000")
 ```
 
 ### Make a Transfer
+Send Kin from a Keypair (as you need the privateKey) to a publicKey on the blockchain.
+
+You can optionally set the commitment level before returning ('Confirmed', 'Finalized' or Processed), the transaction type, or add a referenceId / referenceType. 
```

Comment:
In the future we might want to include also a tiny of example using the referenceId / referenceType, becase we mention the user can add it, but it is not clear how.

Also I noticed this small indentation issue right below, while you are here, maybe you can fix it. :)
![image](https://user-images.githubusercontent.com/1482473/173103841-99caf80a-cc2b-4a8e-99b2-ac6ad5c8fb42.png)